### ISSUE TITLE
wdio-static-server-service not working

### ISSUE THREAD

**Issue by SantoJambit** [webdriverio/webdriverio#5240]
(The readme doesn't specify this project, but since it is included in the packages folder, I'm assuming this is still the right place to report this)

**Environment (please complete the following information):**
 - **WebdriverIO version:**  6.0.8, 6.0.14
 - **Mode:** WDIO Testrunner
 - **If WDIO Testrunner, running sync/async:** sync
 - **Node.js version:** 13.12
 - **NPM version:** 6.14.4
 - **Browser name and version:** chrome 80
 - **Platform name and version:** Ubuntu 18.04
 - **Additional wdio packages used (if applicable):** @wdio/static-server-service,@wdio/cucumber-framework

**Config of WebdriverIO**
You can use the cucumber boilerplate, since it shows the issue.
https://github.com/webdriverio/cucumber-boilerplate

**Describe the bug**
A server is not started. When I run tests, they fail, as the server is not running. When I try to access the server via a browser when the tests are still running, the server is not there.

I've tried it via the cucumber boilerplate and by manually specifying the server like shown in the readme. Same result. I can spawn a server locally with npx serve on the same port without problems.

**To Reproduce**
- clone the cucumber boilerplate and run `npm run test:features`:

https://github.com/webdriverio/cucumber-boilerplate

Not sure if this is a linux-specific issue.

**Expected behavior**
A server should be running.

**Log**
There is nothing in the log, which shows anything about the service. Not even sure it gets started at all.

**Additional context**

> Please use the correct markdown

**Comment by SantoJambit** [comment]
Update: When I configure it like this, there seems to be a non-responding server.. i.e. it just keeps loading:

```
    ['static-server', {
        folders: [
            { mount: '/', path: './demo-app' },
        ],
        port: 8080,
    }],
```

**Comment by kajmagnus** [comment]
Works for me, with WebdriverIO 6.0.15, like so:

```
  services: [
    ... (selenium ... ) ...
     // https://webdriver.io/docs/static-server-service.html
     ['static-server', {
       port: 8080,
       folders: [
         { path: './target/', mount: '/' }]
     }],
```

I didn't know which one of `path` and `mount` referred to the file system, or the browser URL path, but by trial and error, turns out that `path` is a file system path (not a URL path), and `mount` is the URL path where the file system path should appear.

**Comment by christian-bromann** [comment]
I can't reproduce a bug either. @SantoJambit you path is relative which means you select a directory relative from where you run the wdio command from. I recommend using absolute path to be more specific.

**Comment by SantoJambit** [comment]
@kajmagnus and @christian-bromann : It seems the non-responsive server issue I was having with 6.0.8 was fixed in `6.0.13`. The reason I was having issues with 6.0.14 was another:

```
    services: [
        ['chromedriver'], // this will cause an exception, as chromedriver expects options to be present.
        ['static-server', {
            port: 3000,
            folders: [
                { mount: '/', path: './demo-app' },
            ],
        }],
    ],
```

Not sure if this is intended, I would assume, that options of chromedriver service are optional.
This fixes it:
```
    services: [
        ['chromedriver', {}], // empty options to keep chrome driver from throwing
        ['static-server', {
            port: 3000,
            folders: [
                { mount: '/', path: './demo-app' },
            ],
        }],
    ],
```

Also, it seems, that if one service crashes, all other services won't get started either. And this all goes down silently without the user being notified.

**Comment by christian-bromann** [comment]
@SantoJambit thanks for the feedback, I will have a look into the Chromedriver issue.
